### PR TITLE
Sample App FE

### PR DESCRIPTION
**Pull request by micahtaylor**
For https://jira.cms.gov/browse/BB2-797

Screen Shot 2021-08-31 at 5 30 03 PM

### CODE REVIEW

**Review by bfausett77 — APPROVED** [review]
This looks good and I was able to get it up and running locally.  I'll work with you to get this actually pulling data from the sandbox.  Thanks and great work!

**Comment by micahtaylor** [comment]
@nbragdon Good to merge or should we keep this in a working branch and create issues against it?

File: client/src/components/records.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Table, TableCaption, TableRow, TableCell, TableHead, TableBody } from '@cmsgov/design-system';
+
+export default function Records({ }) {   
+    //TODO: Fetch real data 
+    const items = [
+        {
+            id: 1,
```
Comment (nbragdon) [inline client/src/components/records.tsx:7]: Did you reference this data from anywhere? Or just kind of making it up?

File: client/src/components/patient.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import avatar from '../images/patient.png'
+
+export default function Patient() {
+    return (
+        <div>
+            <h3>Clinic records</h3>
+            <div className='ds-u-display--flex ds-u-flex-direction--row ds-u-align-items--center'>
+                <img src={avatar} alt="Profile avatar"/>
+                <ul>
+                    <li>John Doe</li>
```
Comment (nbragdon) [inline client/src/components/patient.tsx:10]: Is this meant to be the application user data, or is this data we should be pulling from blue button?

**Review by nbragdon — APPROVED** [review]
Overall this is looking really good! Just left a few questions but nothing major.

I think the other front end piece of work will basically involve adding in typing.

These components are going to need typed state passed to them most likely.

We can probably just merge this one, and do follow up.

File: client/src/components/records.tsx (hunk 1 above)
Comment (micahtaylor) [reply client/src/components/records.tsx:7]: Randomly pulled items like `"display": "Physician/supplier"` it from [eob testclient](https://sandbox.bluebutton.cms.gov/testclient/ExplanationOfBenefit)

File: client/src/components/patient.tsx (hunk 2 above)
Comment (micahtaylor) [reply client/src/components/patient.tsx:10]: I'd assume application user data as BB doesn't, to my knowledge, have images.

**Pull request merged**